### PR TITLE
Closes #118 : Update package.json dependencies

### LINKED ISSUE #118: webpack: update package.json dependencies
> I think we should upgrade dependencies too (run npm outdated). In particular there is an issue that Karolina encountered with Node 13, node-sass really depends on some Node versions, see here.
>
> With Node 13, node-sass should be at least 4.13.
>
> Maybe we should add a Travis matrix with different Node versions, from LTS (12) to 14?

Left over from #113

See [supported Node.js versions compatibility matrix](https://github.com/sass/node-sass#supported-nodejs-versions-vary-by-release-please-consult-the-releases-page-below-is-a-quick-guide-for-minimum-support)

### PR DESCRIPTION
**Pull request by ParthS007**
closes #118

### CODE REVIEW

**Review by ntarocco — COMMENTED** [review]
Good job! :)
As discussed already IRL, I would have limited the changes to all dependencies to the highest patch and minor versions, without touching the major ones.
IMHO, we will have to re-test all the JS in Invenio if we upgrade the majors. Ping @slint @lnielsen if you have a different opinion.

File: invenio_assets/assets/package.json [hunk 1]
```diff
@@ -17,62 +17,62 @@
...
     "webpack-livereload-plugin": "^2.2.0",
     "webpack-bundle-tracker": "^1.0.0-alpha.1",
-    "webpack-merge": "^4.1.5",
+    "webpack-merge": "^5.1.2",
     "webpack-yam-plugin": "^1.0.1"
   },
   "engines": {
-    "node": ">= 4.0.0",
-    "npm": ">= 3.0.0"
+    "node": ">= 12.18.3",
```
Comment (ntarocco) [inline invenio_assets/assets/package.json:75]:
```suggestion
    "node": ">= 12",
```

File: invenio_assets/assets/package.json [hunk 2]
```diff
@@ -17,62 +17,62 @@
...
+    "npm": ">= 6.14.6"
```
Comment (ntarocco) [inline invenio_assets/assets/package.json:76]:
```suggestion
    "npm": ">= 6"
```

**Comment by slint** [comment]
I'm having a go at using these bumps to run the Webpack build. Will update with any findings here:

-  `CleanWebpackPlugin` has changed import paths

**Comment by slint** [comment]
Tested along with https://github.com/inveniosoftware/invenio-previewer/pull/125 and everything works as expected

**Pull request merged**